pkg/sql/schemachanger/scpb/scpb.proto

```diff
@@ -53,6 +53,10 @@ enum Status {
   // Intermediate states on the transient index dropping path
   TRANSIENT_DELETE_ONLY = 13;
   TRANSIENT_WRITE_ONLY = 14;
+
+  // Intermediate states on check constraint dropping and adding path.
+  Check_Constraint_Validating = 16;
+  Check_Constraint_Validated = 17;
```

Comment:
We'll need similar states for other constraints so I'm not sure I like these very much. Can we reuse existing states? What about reusing VALIDATED, and using WRITE_ONLY for "validating"? From what I understand, in that state, we apply the constraint only on stuff we write to the table. Please correct me if I'm wrong.